app/src/main/res/values/strings.xml

```diff
@@ -407,6 +407,7 @@
   <string name="reveal">Reveal</string>
   <!-- TextViewBindingAdapters -->
   <string name="just_now">just now</string>
+  <string name="recently">recently</string>
```

Comment:
The name ought to be clearer (the other names aren't very clear). Perhaps: ``last_logged_in_recently`` would be clearer?

Reply:
Changed.